docs/api/aggregate/command-handler.md

```diff
@@ -3,6 +3,15 @@ id: command-handler
 title: Command Handler
 ---
 
+:::info TypeScript Support
+
+Command handlers are grouped into an Aggregate container object that has an associated TypeScript type:
+
+- Type Name - `Aggregate`
+- Package - `@resolve-js/core`
+
+:::
+
 A command handler function has the following structure:
 
```

Comment:
I have a question regarding the description on line 33 below.

`The aggregate version that is a number incremented for each consequent event with the current aggregateId.`

What do you mean by "consequent event"? For example, do you mean "subsequent event"?

See the following topic for an explanation of _consequent_ vs. _subsequent_:
https://www.merriam-webster.com/dictionary/consequent#note-1

Reply:
Thank you for the link. Yes, "subsequent" is the right word. I fixed this error here and in the `event.md` file.

Reply:
Thank you.